repos/system_upgrade/el7toel8/actors/addupgradebootentry/libraries/library.py

```diff
@@ -30,3 +30,21 @@ def get_boot_file_paths():
                                       details={'details': 'Did not receive a message about the leapp-provided'
                                                           'kernel and initramfs'})
     return boot_content.kernel_path, boot_content.initram_path
+
+
+def fix_grub_config_error():
+    lines = []
```

Comment:
You don't need to initialize it when you set it later by `lines = f.readlines()`.